.github/workflows/camel_version.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        camel-version: [ "4.0.0",  "4.0.0.redhat-00036" ]
+        camel-version: [ "4.4.0",  "4.0.0.redhat-00036", "4.4.0.redhat-00019"]
```

Comment:
do we still need to test also with `"4.0.0.redhat-00036"` or 4.4.0 upstream/productized is enough?

Reply:
see https://github.com/camel-tooling/camel-dap-client-vscode/pull/665#discussion_r1629540680